code/src/gfx.c

```diff
@@ -1052,7 +1052,7 @@ void Gfx_Update(void) {
         Gfx_ShowMultiplayerSyncMenu();
     }
 
-    // The update is called here so it works while in file select
+    // The update is called here so it works while in different game modes (title screen, file select, boss challenge, credits, MQ unlock)

```

Comment:
The `Multiplayer_Update` below here is no longer called while in the boss select menu, which can cause syncs to fail.

Reply:
Thanks, I had misplaced the cutscene check in IsInGame, oops.

I changed the check to send ghost data but I haven't tested it yet. I'll do it later if you don't do it in the mean time

Reply:
I think everything works now. Let me know if there are still issues